apps/happy-blocks/src/support-page/embed-placeholder.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Placeholder } from '@wordpress/components';
+import { _x } from '@wordpress/i18n';
+import classnames from 'classnames';
+import './edit.scss';
+
+type EmbedPlaceHolderProps = {
+	icon: JSX.Element;
+	label: string;
+	className: string;
+	instructions: string;
+	notices?: JSX.Element | undefined;
+	onSubmit(): void;
+	placeholder: string;
+	url: string;
+	updateUrl( s: string ): void;
+};
+
+/**
+ * UI for configuring the embed
+ */
+export const EmbedPlaceHolder = ( props: EmbedPlaceHolderProps ) => {
+	return (
+		<div className={ classnames( 'hb-support-page-embed-placeholder', props.className ) }>
```

Comment:
I see that we have a different classname prefix above in `wp-block-p2-embed`. Could we unify these, please?